luigi_pipeline/lib/hail_vep_runners.py

```diff
@@ -559,5 +559,7 @@ class HailVEPDummyRunner(HailVEPRunnerBase):
            'variant_class': 'SNV'},)
 
 
-    def run(self, mt, genome_version):
+    def run(self, mt, genome_version, vep_config_json_path=None):
+        if isinstance(mt, hl.Table):
+            return mt.annotate(vep=self.MOCK_VEP_DATA)
         return mt.annotate_rows(vep=self.MOCK_VEP_DATA)
```

Comment:
Is there any case in the new pipeline that we need to annotate vep on a MatrixTable, instead of a regular table?

Reply:
I think at this point no!  So this code can be reverted.